python/ray/worker.py

```diff
@@ -620,6 +622,10 @@ def init(
             specify a specific node address. If the environment variable
             `RAY_ADDRESS` is defined and the address is None or "auto", Ray
             will set `address` to `RAY_ADDRESS`.
+            Addresses can be prefixed with a protocol to connect using Ray
+            Client. For example, passing in the address
+            "ray://123.45.67.89:50005" will attempt to start a ray client
+            session through the specified address.
```

Comment:
`start a ray client session` is a bit confusing, we should say `connect to the cluster at the given address using the Ray client`.